vendor/swaggen/models/dependencies_list.go

```diff
@@ -15,7 +15,7 @@ import (
 type DependenciesList struct {
```

Comment:
Is this file an auto generated one. If so we should remove it from source control.